package/kernel/rtl8812au-ct/Makefile

```diff
@@ -41,6 +41,7 @@ NOSTDINC_FLAGS := \
 	-I$(STAGING_DIR)/usr/include/mac80211/uapi \
 	-include backport/backport.h \
 	-Wno-error=address \
+	-Wno-error=missing-prototypes \
```

Comment:
maybe this driver can be removed, mac80211 have native support for it now (kmod-rtw88-8812au).
Cc: @hauke

Reply:
Maybe for some use cases this older driver works better, but I would not invest too much time into it. We should prefer the driver form the mac80211 backports.